src/modules/dataman/dataman.c

```diff
@@ -949,7 +949,15 @@ _file_initialize(unsigned max_offset)
 		close(dm_operations_data.file.fd);
 
 		if (incompat) {
+#ifdef CONFIG_ARCH_BOARD_AEROCORE2
```

Comment:
@davids5 

Thanks for the tips on SDCARD and LED stuff.  I didn't notice the tone alarm for the SDCARD before.  I guess since it's always been there I always just got used to it.  That should be cleared now.  Also nice touch with the LEDs.  We weren't doing much with them so that's very helpful.  We only have 2 LEDs accessible to PX4 (other one connected to co-processor).  From what I can tell, the GREEN LED has more functionality so I've elected to drop the BLUE LED functionality and just go with GREEN.

Everything added and pushed.  Thanks.